src/assets/scss/_give.scss

```diff
@@ -629,6 +629,32 @@ input[type="radio"] {
 }
 /* Product Config Form Amounts End */
 
+.radio-method,
```

Comment:
This seems to be the main change in this commit, so the commit message should reflect what is happening here.